login_page.py

```diff
@@ -0,0 +1,32 @@
+"""Login page"""
+"""Importing modules"""
+from tkinter import *
+import tkinter.font as font
+from PIL import ImageTk, Image
+
+root=Tk()
+root.geometry("200x200+600+200")
+root.title("Walk2Zero")
+"""Bckground image"""
+img = ImageTk.PhotoImage(Image.open("walk2zeo2.jpg"))
+label_image=Label(root, image=img)
+label_image.place(x=0,y=0,relwidth=1,relheight=1)
+
+"""Top Heading"""
+

```

Comment:
Again, only a small thing and nothing to do with functionality. But we should be consistent with line spacing and make sure we are adding blank lines between the different sections of code in a consistent way as it makes it more readable.

For example, here we have a blank line before and after the Top Heading comment. But later on there are no blank lines around the Second heading-login comment.